components/UniversalGlossary.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="universal-glossary__container">
+    <bx-data-table>
+      <bx-table>
+        <bx-table-head>
+          <bx-table-header-row class="universal-glossary__table-header">
```

Comment:
```suggestion
          <bx-table-header-row class="universal-glossary__header">
```
I think it is more representation agnostic.